src/pages/MainPage.vue

```diff
@@ -521,8 +521,8 @@ export default {
     changeActiveRoad: function (event) {
       this.activeRoad = event;
     },
-    deleteRoad: function (roadID) {
-      Vue.delete(this.roads, roadID);
+    devareRoad: function (roadID) {
```

Comment:
There's a lot of this `delete` being replaced by `devare`. That probably should be fixed.